db/schema.rb

```diff
@@ -606,6 +608,19 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "session_blocked_lists", force: :cascade do |t|
+    t.string "devise_session"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "sessions_blacklists", force: :cascade do |t|
+    t.string "sessionid", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["sessionid"], name: "index_sessions_blacklists_on_sessionid"
+  end
+
```

Comment:
what happened here?

Reply:
I will update the schema. It is picking some other changes.